README.md

```diff
@@ -4,6 +4,8 @@ If you are unfamiliar with PerfView, there are [PerfView video tutorials](http:/
 As well as [Vance Morrison's blog](http://blogs.msdn.com/b/vancem/archive/tags/perfview) which also gives overview and getting 
 started information. 
 
+[![Build status](https://ci.appveyor.com/api/projects/status/fxtu3xa874whk2w0?svg=true)](https://ci.appveyor.com/project/sharwell/perfview)
```

Comment:
Put a small description before this:  e.g.  Perfview uses appveyor to produce automated builds (and eventually tests?).  We will also probably need a new .md file that talks quickly about the 'high level' view of this build / test system (where the yml file is and logically what it is doing, and security issues people need to know about, links to appVeyor, what the rules are (free for open source?). etc).  

Along those lines:  what happens with forks?   How does security work?